subsystem/src/main/java/org/wildfly/extension/grpc/GrpcSubsystemParser_1_0.java

```diff
@@ -31,4 +50,106 @@ public class GrpcSubsystemParser_1_0 extends PersistentResourceXMLParser {
     public PersistentResourceXMLDescription getParserDescription() {
         return xmlDescription;
     }
+
+    @Override
```

Comment:
All of the below should be deletable with a `PersistentResourceXMLParser`.

Reply:
> All of the below should be deletable with a PersistentResourceXMLParser.

I see that PersistentResourceXMLParser is mentioned in "Extending Wildfly", so there's a minimal chance that I had a good reason for not using PersistentResourceXMLParser. More likely, it's just ignorance. I'll look at it.

Reply:
The `PersistentResourceXMLParser` is the "new" way of doing subsystems. I will admit I have NOT used it yet, but I looked at other subsystems that do which was the reason I think the below was not needed. I'm definitely happy to help look though, so feel free to ping me on chat. I'd like to learn how to use it as well :)

Reply:
Ok, thanks, @jamezp. Reading about PersistentResourceXMLParser will be my next step. Along with deleting the tabs. ;-)

Reply:
Ok, I see:
```
    static {
        xmlDescription = builder(GrpcSubsystemDefinition.INSTANCE.getPathElement(), GrpcSubsystemSchema.GRPC_1_0.getUriString())
                .addAttributes(
                        GrpcAttribute.WILDFLY_GRPC_SERVER_PORT,
                        GrpcAttribute.WILDFLY_GRPC_SERVER_HOST)
                .build();
    }
```
is all you need in GrpcSubsystemParser_1_0. Wow!

Reply:
Yeah, it's pretty nice eh?

Reply:
Fixed